tests/test_optimizely_config.py

```diff
@@ -1800,3 +1802,46 @@ def test_get_delivery_rules(self):
                 self.assertIsInstance(delivery_rule, optimizely_config.OptimizelyExperiment)
 
         self.assertEqual(expected_features_map_dict, actual_features_map_dict)
+
```

Comment:
we can remove these tests for optimizely_config

Reply:
We can move these tests to project_config tests.

Reply:
done